### ISSUE TITLE
Error in Mavericks: "only use PostScript names"

### ISSUE THREAD

**Issue by leonelgalan** [voormedia/rails-erd#61]
``` bash
brew info graphviz
graphviz: stable 2.34.0, devel 2.35.20130914.0446
...
```

Errors are shown one by one on this order:

```
Saving diagram failed!
Graphviz produced errors. Verify it has support for filetype=pdf, or use filetype=dot.
Original error: 2013-10-30 00:54:38.917 dot[42907:d07] CoreText performance note: Client called CTFontCreateWithName() using name "Arial Italic" and got font with PostScript name "Arial-ItalicMT". For best performance, only use PostScript names when calling this API.
...
Saving diagram failed!
Graphviz produced errors. Verify it has support for filetype=pdf, or use filetype=dot.
Original error: 2013-10-30 00:56:52.537 dot[43000:d07] CoreText performance note: Client called CTFontCreateWithName() using name "Arial" and got font with PostScript name "ArialMT". For best performance, only use PostScript names when calling this API.
...
Saving diagram failed!
Graphviz produced errors. Verify it has support for filetype=pdf, or use filetype=dot.
Original error: 2013-10-30 00:57:12.159 dot[43036:d07] CoreText performance note: Client called CTFontCreateWithName() using name "Arial Bold" and got font with PostScript name "Arial-BoldMT". For best performance, only use PostScript names when calling this API.
```

Replacing the font names with the PostScript names solves the issue. I'm not sure if this is backwards compatible, so I'm not pushing my code. I will, gladly, once this is discussed.

**Comment by karinelisabeth** [comment]
Hi, I have tried the solutions above, eg: brew reinstall graphviz --with-bindings --with-freetype --with-librsvg --with-pangocairo  as suggested by @maciejkowalski & the changes to gem files proposed by @hayduke19us , but none work for me I still get the post-script error... I can't run the 'mavericks' version from @paulwittmann as it doesn't seem to support rails v4.2? (I am currently running the modified rake erb version posted in https://github.com/voormedia/rails-erd/issues/85, github link: https://github.com/ready4god2513/rails-erd/tree/rails-4.2-support-fix.) Any suggestions what to try next...? thanks!

**Comment by kerrizor** [comment]
I just cut a v1.3.0 of this gem that should fix font issues /and/ support Rails 4.2
